server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationRelocationIT.java

```diff
@@ -395,4 +398,129 @@ public void testRelocateWithQueuedOperationsDuringHandoff() throws Exception {
         waitForSearchableDocs(totalDocCount, replica, newPrimary);
         verifyStoreContent();
     }
+
+    /**
+     * This test verifies that adding a new node which results in peer recovery as replica; also bring replica's
+     * replication checkpoint upto the primary's by performing a round of segment replication.
+     */
+    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/5669")
+    public void testNewlyAddedReplicaIsUpdated() {
+        internalCluster().startNode(featureFlagSettings());
```

Comment:
nit - you shouldn't need to pass featureFlags anymore if overriding featureFlagSettings method.